lib/tasks/prepare_postgres.rake

```diff
@@ -48,13 +55,27 @@ private
     end
   end
 
-  def package_missing?(package)
-    command = 'which ' + package
-
-    `#{command}`[0] != '/'
+  def exec_present? name
+    system("which #{name} 2>&1 >/dev/null")
   end
 
   def darwin_install_package(package)
```

Comment:
As we're moving away from a `one-click-installer`, let's delete this method to not have it slushing around perhaps?